addons/web/static/lib/bootstrap/scss/_utilities.scss

```diff
@@ -709,7 +709,7 @@ $utilities: map-merge(
     // scss-docs-end utils-interaction
     // scss-docs-start utils-border-radius
     "rounded": (
-      property: border-radius,
+      property: --box-border-radius,
```

Comment:
You cannot change the lib file directly. You can change this map by modifying it at the right place: I see we have a `utilities_custom` stuff, it should probably be a specific one for the frontend, in web_editor (or in the future html_builder I guess).